src/nanoFramework.Graphics/TouchPanel/Core/TouchPanel.cpp

```
@@ -162,10 +162,10 @@ HRESULT TouchPanelDriver::GetDeviceCaps(unsigned int iIndex, void *lpOutput)
             return (g_TouchPanelDriver.CalibrationPointGet((TOUCH_PANEL_CALIBRATION_POINT *)lpOutput));
 
         default:
-            return FALSE;
+            return false;
```

Clang (14.0.6)

[warning] implicit conversion bool -> 'HRESULT' (aka 'int')

(readability-implicit-bool-conversion)